src/nvim/eval.c

```diff
@@ -9595,13 +9595,15 @@ static void f_getcwd(typval_T *argvars, typval_T *rettv, FunPtr fptr)
       if (from) {
         break;
       }
-    case kCdScopeTab:         // FALLTHROUGH
+      // fallthrough
```

Comment:
I read @ZyX-I's [comment](https://github.com/neovim/neovim/issues/6512#issuecomment-293977487) to mean "use comments except where an attribute is needed", but I could have misinterpreted it.  I don't have a strong opinion either way.